userscript/origin.user.js

```diff
@@ -19,8 +19,16 @@
     var subdomain = domain_parent.endsWith('suse.de') ? 'tortuga' : 'operator';
     var url = 'https://' + subdomain + '.' + domain_parent + '/origin/package/' + project + '/' + package;
 
+    $('ul.clean_list, ul.list-unstyled').append('<li id="osrt-origin"><i class="fas fa-spinner fa-spin text-info"></i> loading origin...</li>');
```

Comment:
The icon does not work in old theme (which is the reason for the two selectors). Otherwise works properly. Given the old theme does not use an icon theme and uses cached icon URLs it is a bit cumbers to utilize from the outside like this. The new theme makes it trivial. I am fine with leaving like this, but just making you aware.

Reply:
Would be an argument for adding back "origin: " prefix in the case if one cares about it. As otherwise it is rather context-less.

Old theme:
![image](https://user-images.githubusercontent.com/22943929/57147670-bc640a80-6d8d-11e9-8a4b-a9a21ff3d05b.png)

New theme:
![image](https://user-images.githubusercontent.com/22943929/57147692-c84fcc80-6d8d-11e9-825d-6633117c6377.png)